eng/SharedFramework.Local.props

```diff
@@ -47,6 +47,7 @@
     <AspNetCoreAppReference Include="Microsoft.AspNetCore.Server.Kestrel" />
     <AspNetCoreAppReference Include="Microsoft.AspNetCore.Server.Kestrel.Transport.Abstractions" />
     <AspNetCoreAppReference Include="Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets" />
+    <AspNetCoreAppReference Include="Microsoft.AspNetCore.Authentication.Certificate" />
```

Comment:
I just ran `GenerateProjectList.ps1` these changes came from that auto'magically'

Reply:
I assume that's because of IsAspNetCoreApp in the csproj.
https://github.com/aspnet/AspNetCore/pull/9756/files#diff-f9b22fb425e6bb8b26f9464265f51ea2R6

Going into the shared framework needs to be an explicit decision. @anurse @blowdart

Reply:
Thanks for catching this, Chris. Please review these requirements as a part of this discussion: https://github.com/aspnet/AspNetCore/blob/master/docs/SharedFramework.md#what-goes-into-the-shared-framework

Reply:
Doesn't look like any of the other authentication schemes are in the shared framework including cookies, so seems like this would be a no, removing this for now unless @blowdart really does want this included

Reply:
Never mind, cookies is there, I'll leave this for now until we figure out whether it belongs

Reply:
No, it doesn't.

Reply:
Ok removed from shared framework